js/app/test/file_component_events.spec.ts

```diff
@@ -49,3 +49,13 @@ test("File component properly handles drag and drop of image and video files.",
 		page.getByLabel("# Load Upload Multiple Files Image/Video")
 	).toHaveValue("2");
 });
+
+test("File component properly handles drag and drop of pdf file.", async ({
```

Comment:
@abidlabs Already added this test. Did you mean another js specific unit test instead?

Reply:
Added a new test for invalid file types.